tests/unit/test_benchmark.py

```diff
@@ -176,3 +178,43 @@ def test_progress_bar_updates(tqdm_mock):
 
     # Assert
     tqdm_mock.assert_called_once_with(ANY, total=1, position=0, leave=True)
+
+
+def test_benchmark_single_table_custom_synthesizer():
```

Comment:
this seems more like an integration test to me since it will be downloading the dataset and metadata behind the scenes

Reply:
Yeah, I felt so as well, but I really wanted to get that extra 11% coverage...

Reply:
Feel free to add more unit tests 😂 . If you do though, we should mock any public or external functions being used